tests/TestCTimers.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
```

Comment:
Please add the lines:

// SPDX-License-Identifier: GPL-2.0-or-later
// Copyright (C) 2023 The MMapper Authors

Reply:
Fixed